tokio-macros/src/entry.rs

```diff
@@ -326,11 +334,11 @@ fn parse_knobs(
             #rt
                 .enable_all()
                 .build()
-                .unwrap()
+                .expect("Failed building the Runtime")
                 .block_on(async #body)
```

Comment:
As an aside, only this `unwrap` appears in the generated output of the macro. The rest of the unwraps are unrelated to #3887.